conf/messages.en

```diff
@@ -338,9 +338,10 @@ alreadyRegistered.account.link.text = your Soft Drinks Industry Levy account.
 
 enterBusinessDetails.title = Enter your business details
 enterBusinessDetails.heading = Enter your business details
-enterBusinessDetails.subtext = This will help us match your details against information we currently hold.
+enterBusinessDetails.subtext1 = This will help us match your details against information we currently hold.
+enterBusinessDetails.subtext2 = You will need to enter your Corporation Tax or company Unique Taxpayer Reference number (UTR). You can <a href={0} target="_blank">find a lost UTR (opens in new tab).</a>
```

Comment:
This should be: 
You can <a class='govuk-link' href={0} target="_blank">find a lost UTR (opens in new tab).</a>

Reply:
it isn't showing, sorry:  Please add class='govuk-link' to the <a href